### PR TITLE
Increase ulimits as suggested by mongo docs.

### PR DESCRIPTION
**Pull request by znep**

### CODE REVIEW

File: templates/default/mongod.service.erb [hunk 1]
```diff
@@ -12,6 +12,8 @@ PIDFile=/var/run/mongod/mongod.pid
 ExecStart=/usr/bin/mongod -f /etc/mongod.conf --pidfilepath /var/run/mongod/mongod.pid --fork
 TimeoutStopSec=5
 KillMode=mixed
+LimitNOFILE=64000
+LimitNPROC=64000
```
Comment (saraid) [inline templates/default/mongod.service.erb:16]: 64000 processes seems like a lot?

Comment (znep) [reply templates/default/mongod.service.erb:16]: It is actually threads on linux, and it is a lot but that is what the docs suggest and far be it from me to argue.

**Review by saraid — APPROVED** [review]
I think allowing too many processes is smelly, but there's not likely anything problematic with doing so, so I'll approve.

File: templates/default/mongod.service.erb (hunk 1 above)
Comment (saraid) [reply templates/default/mongod.service.erb:16]: 🤷

**Pull request merged**